.github/workflows/mkdocs-deploy.yml

```
@@ -15,8 +13,11 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.x
-      - run: pip install -r docs/requirements.txt
-      - run: |
+      - name: Install python action for doc extraction
+        run: |
+          pip install . -r docs/requirements.txt
```

This should be a 1 line cmd
```suggestion
        run: pip install . -r docs/requirements.txt
```